packages/pin-input/src/pin-input.tsx

```diff
@@ -60,7 +60,7 @@ if (__DEV__) {
 export interface PinInputFieldProps extends PropsOf<typeof chakra.input> {}
 
 export const PinInputField = forwardRef<PinInputFieldProps, "input">(
-  function PinInputField(props, ref) {
+  function PinInputField(props, ref: any) {
```

Comment:
that coincidentally crept in I assume? :)

Reply:
I have no idea, the ref type is the same as before but now it gives a weird error, I tried using RefObject, Reg, MutableRef, AssignableRef but nothing works